### ISSUE TITLE
Message ID counting is wrong

### ISSUE THREAD

**Issue by thatoddmailbox** [MyHomeworkSpace/PlanBot#1]
It declares "TRIIIIIIIIIPS" and "QUAAAAAADS" way too frequently. I think Slack changed how the message IDs work and they aren't super sequential anymore. (though I'm not sure they ever were really sequential...) I'm also not sure if there's a better way to track the messages. I guess we could have our own counter, but that seems like it wouldn't work super well?

I am also impressed that this somehow still works with the current Slack API even though the last commit was 3 years ago.

tl;dr @willweinstein is bad at programming
